RELEASE_NOTES.md

```diff
@@ -1,5 +1,521 @@
+
+### windows_security_policy
+
+Use the `windows_security_policy` resource to modify location security policies on Windows hosts. See the [windows_security_policy documentation](https://docs.chef.io/resources/windows_security_policy/) for full usage information.
+
+### windows_user_privilege
+
+Use the `windows_user_privilege` resource to add users and groups to the specified privileges on Windows hosts. See the [windows_user_privilege documentation](https://docs.chef.io/resources/windows_user_privilege/) for full usage information.
+
+## Improved Resources
+
+### compile_time on all resources
+
+The `compile_time` property is now available for all resources so that they can be set to run at compile time without the need forcing the action.
+
+This allows you replace forcing resources to run at compile time:
+
+```ruby
+  my_resource "foo" do
+    action :nothing
+  end.run_action(:run)
+```
+
+With the simpler compile_time property:
+
+```ruby
+  my_resource "foo" do
+    compile_time true
+  end
+```
+
+### build_essential
+
+The `build_essential` resource includes a new `:upgrade` action for macOS systems that allows you to install updates to the Xcode Command Line Tools available via Software Update.
+
+### cron
+
+The `cron` resource has been updated to use the same property validation for cron times that the `cron_d` resource uses. This improves failure messages when invalid inputs are set and also allows for `jan`-`dec` values to be used in the `month` property.
+
+### dnf_package
+
+The `dnf_package` resource, which provides `package` under the hood on any system shipping with DNF, has been greatly refactored to resolve multiple issues. The version behavior and overall resource capabilities now match that of the `yum_package` resource.
+
+- The `:lock` action now works on RHEL 8
+- Fixes to prevent attempting to install the same package during each Chef Infra Client run.
+- Resolved several idempotency issues.
+- Resolved an issue where installing a package with `options '--enablerepo=foo'` would fail.
+
+### git
+
+The `git` resource now fully supports why-run mode and no longer checks out the `deploy` branch by default as mentioned in the breaking changes section.
+
+### locale
+
+The locale resource now supports setting the system locale on Windows hosts.
+
+### msu_package resource improvements
+
+The `msu_package` resource has been improved to work better with Microsoft's cumulative update packages. Newer releases of these packages will not correctly install over previous versions. We've also extended the default timeout for installing MSU packages to 60 minutes. Thanks for reporting the timeout issue [@danielfloyd](https://github.com/danielfloyd)
```

Comment:
Is this sentence correct? I ask because I did a double take when reading it.
>Newer releases of these packages will not correctly install over previous versions.

Reply:
Check it now.